runtime/fundamentals/testing.md

```diff
@@ -588,3 +588,51 @@ Snapshot testing enables catching a wide array of bugs with very little code. It
 is particularly helpful in situations where it is difficult to precisely express
 what should be asserted, without requiring a prohibitive amount of code, or
 where the assertions a test makes are expected to change often.
+
+## Managing Permissions in Tests
+
+When running tests in Deno, it's important to understand that the `permissions`
+object in test configuration doesn't grant permissions to the test. Instead, it
+specifies which permissions the test requires to run correctly.
+
+A test cannot have more permissions than were granted to the Deno process itself
+when it was started (e.g., with `deno test --allow-read`). This security model
+prevents JavaScript from breaking out of its sandbox, which could create
+security vulnerabilities.
+
+For example, if you run `deno test` without any permission flags, tests that
+specify permissions will still be denied those permissions. You must explicitly
+run the test command with the necessary permission flags.
+
+You can explicitly limit permissions with the permissions object:
```

Comment:
I don't know the backstory to this: But I'm kinda wondering how we're selling this overall feature. Like when would you want to add the `permissions` option to a test and when not?